api/v3/Payment.php

```diff
@@ -52,18 +49,36 @@ function civicrm_api3_payment_get($params) {
     if (!empty($eftIds)) {
       $ftParams = [
         'id' => ['IN' => $eftIds],
-        'is_payment' => 1,
+        'is_payment' => $params['is_payment'],
```

Comment:
I don't think this change makes sense - the point of the payment api is to retrieve payment financial trxns (payments or refunds or potentially failed payments)

Reply:
Note there IS a financial_trxn api for non-payment records